metrics.go

```diff
@@ -3,27 +3,57 @@
 package goa
 
 import (
+	"regexp"
+	"strings"
 	"time"
 
 	"github.com/armon/go-metrics"
 )
 
-// metriks is the local instance of metrics.Metrics
-var metriks *metrics.Metrics
+const (
+	allMatcher      string = "*/*"
+	allReplacement  string = "all"
+	normalizedToken string = "_"
+)
+
+var (
+	// interface for metrics.Metrics
+	metriks Metrics
+
+	// used for normalizing names by matching '*' and '/' so they can be replaced.
+	invalidCharactersRE = regexp.MustCompile(`[\*/]`)
```

Comment:
should the regexp be the same as the one in the prometheus package? the test should be whether the character does *not* match the regexp (white list instead of black list).